src/Dev/SapphireTest.php

```diff
@@ -70,6 +72,16 @@ class SapphireTest extends PHPUnit_Framework_TestCase
      */
     protected $fixtureFactory;
 
+    protected $backupGlobalsBlacklist = array(
+        'injector',
```

Comment:
Any variable declared in the global scope: http://php.net/manual/en/reserved.variables.globals.php

In PHP all variables that aren't declared in functions/classes are automatically in the global scope. You can access these vars using the `global` keyword.

On another point, there is a risk that modules that declare vars in the global scope will then have to construct their own blacklist - so maybe we create an API for that?